mdlinkcheckconfig.json

```diff
@@ -38,6 +52,31 @@
     {
       "pattern": "https://ospo.library.jhu.edu/",
       "reason": "500"
+    },
+    {
+      "pattern": "https://www.heliosopen.org/members",
+      "reason": "403"
+    },
+    {
+      "pattern": "https://you.stonybrook.edu/mckinnonrosati/open-source/",
+      "reason": "0"
```

Comment:
Remove this block. You don't add this link in the current PR, so this is unnecessary in this one.